### ISSUE TITLE
[Bug]:   Deploy cubefs  on k8s  server pod  error

### ISSUE THREAD

**Issue by 820678105** [cubefs/cubefs#2565]
### Contact Details

820678105@qq.com

### Is there an existing issue for this?

- [X] I have searched all the existing issues

### Priority

low (Default)

### Environment

```markdown
- CubeFS version: v3.2.0
- Deployment mode(docker or standalone or cluster): k8s cluster
- Dependent components:
- OS kernel version(Ubuntu or CentOS): CentOS 7,9
- CPU/Memory:
- Others:
```

### Current Behavior

master pod   error

### CubeFS Log

Defaulted container "master-pod" out of: master-pod, init-config (init)
{
"role": "master",
"ip": "10.65.220.13",
"listen": "17010",
"prof": "17020",
"id": "1",
"peers": "1:master-0.master-service:17010",
"retainLogs": "2000",
"logDir": "/cfs/logs",
"logLevel": "error",
"walDir": "/cfs/data/master/raft",
"storeDir": "/cfs/data/master/rocksdbstore",
"consulAddr": "http://consul-service:8500/",
"exporterPort": "9500",
"clusterName": "my-cluster",
"metaNodeReservedMem": "67108864"
}
start master
/cfs/bin/cfs-server: /lib64/libstdc++.so.6: version GLIBCXX_3.4.20' not found (required by /cfs/bin/cfs-server) /cfs/bin/cfs-server: /lib64/libstdc++.so.6: version CXXABI_1.3.8' not found (required by /cfs/bin/cfs-server)
/cfs/bin/cfs-server: /lib64/libstdc++.so.6: version GLIBCXX_3.4.22' not found (required by /cfs/bin/cfs-server) /cfs/bin/cfs-server: /lib64/libstdc++.so.6: version GLIBCXX_3.4.21' not found (required by /cfs/bin/cfs-server)
./cfs-server: /lib64/libstdc++.so.6: version GLIBCXX_3.4.20' not found (required by ./cfs-server) ./cfs-server: /lib64/libstdc++.so.6: version CXXABI_1.3.8' not found (required by ./cfs-server)
./cfs-server: /lib64/libstdc++.so.6: version GLIBCXX_3.4.22' not found (required by ./cfs-server) ./cfs-server: /lib64/libstdc++.so.6: version GLIBCXX_3.4.21' not found (required by ./cfs-server)

**Comment by 820678105** [comment]
> Can you execute `strings /usr/lib64/libstdc++.so.6 | grep GLIBC`, and tell us the output?

i  am using  the  offical  image:  cubefs/cfs-server:v3.2.0  ,  when i  run this  image and execute the command ,  as you expect , GLIBC  version is old, maybe because of  old gcc version.  Does  the offical image  you  offered  have problem ? 
this is the  output:
strings /usr/lib64/libstdc++.so.6 | grep GLIBC
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBC_2.3
GLIBC_2.2.5
GLIBC_2.14
GLIBC_2.4
GLIBC_2.3.2
GLIBCXX_DEBUG_MESSAGE_LENGTH

**Comment by NaturalSelect** [comment]
> > Can you execute `strings /usr/lib64/libstdc++.so.6 | grep GLIBC`, and tell us the output?
> 
> i am using the offical image: cubefs/cfs-server:v3.2.0 , when i run this image and execute the command , as you expect , GLIBC version is old, maybe because of old gcc version. Does the offical image you offered have problem ? this is the output: strings /usr/lib64/libstdc++.so.6 | grep GLIBC GLIBCXX_3.4 GLIBCXX_3.4.1 GLIBCXX_3.4.2 GLIBCXX_3.4.3 GLIBCXX_3.4.4 GLIBCXX_3.4.5 GLIBCXX_3.4.6 GLIBCXX_3.4.7 GLIBCXX_3.4.8 GLIBCXX_3.4.9 GLIBCXX_3.4.10 GLIBCXX_3.4.11 GLIBCXX_3.4.12 GLIBCXX_3.4.13 GLIBCXX_3.4.14 GLIBCXX_3.4.15 GLIBCXX_3.4.16 GLIBCXX_3.4.17 GLIBCXX_3.4.18 GLIBCXX_3.4.19 GLIBC_2.3 GLIBC_2.2.5 GLIBC_2.14 GLIBC_2.4 GLIBC_2.3.2 GLIBCXX_DEBUG_MESSAGE_LENGTH

Can you install new version of GLIBC, and try again?

BTW, I recommend you to use `3.3.0`.

**Comment by 820678105** [comment]
> 
yes,  i am trying  to install newer  version of   gcc  in  the  image.     it is  difficult to  build an image from source code by myself,   if  the  official  image  v3.3.0  is available,  i am glad to  use.

**Comment by baijiaruo** [comment]
@820678105

**Comment by SteveWang63** [comment]
I have the same problem with the new version. Is there a solution?

**Comment by LonelysChen** [comment]
I solved this problem using version 3.3.0 of the image

**Comment by Huweicai** [comment]
helm default cfs version upgrade to v3.3.0
